docs/index.md

```diff
@@ -41,12 +42,10 @@ firmware.
 
 ## Supporting Fluidd
 
-Fluidd is developed and maintained by [Craig Bassett](https://github.com/cadriel)
-and our contributors - along with support from the entire Klipper Team, associated printer communities and you, the user.
-
-- [Make a recurring payment with Patreon](https://patreon.com/cadriel)
-- [Make a one time payment with Ko-Fi](https://ko-fi.com/cadriel)
-- [Make a one time payment with Paypal](https://paypal.me/fluiddui)
+Fluidd was created by [Craig Bassett](https://github.com/cadriel) and is currently
+developed and maintained by [Fluidd Maintainers](https://github.com/orgs/fluidd-core/teams/maintainers)
```

Comment:
I don't think org team members are accessible publicly. Maybe this works?
```suggestion
developed and maintained by the [Fluidd team](https://github.com/orgs/fluidd-core/people)
```

Reply:
Good point, I actually thought that was public, I see now it isn't... yours is better! 🙂